src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/HubProtocolExtensions.cs

```diff
@@ -8,6 +8,12 @@ namespace Microsoft.AspNetCore.SignalR.Internal.Protocol
     public static class HubProtocolExtensions
     {
         public static byte[] WriteToArray(this IHubProtocol hubProtocol, HubMessage message)
```

Comment:
We shouldn't need both now. This wasn't just used in testing, it's also used by the `SerializedHubMessage` class to store pre-serialized versions: https://github.com/aspnet/SignalR/blob/01089da84d5d55afc7b1cb3983d8660ef4e7ae12/src/Microsoft.AspNetCore.SignalR.Core/Internal/SerializedHubMessage.cs#L68

But we shouldn't need both, they're the same behavior.

Reply:
Done